### PR TITLE
FB Automate Building API Locally

### PR DESCRIPTION
**Pull request by gurps-singh**
This PR makes changes to the `docker-compose.yml` file to use the `licenses-db` container which will now house the database. This removes the need for setting your own database locally. There are tweaks to the `run-local.sh` script to automate the building of the API for local development and a tidy up of the README to match the changes.

### CODE REVIEW

**Review by SebNegahbanMoJ — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -1,17 +1,23 @@
 version: "3"
 services:
-  create-and-vary-a-licence-api:
-    build:
-      context: .
-    network_mode: "host"
-    container_name: create-and-vary-a-licence-api
+  licences-db:
+    image: postgres
```
Comment (andrewrlee) [inline docker-compose.yml:4]: good to specify the [version](https://github.com/ministryofjustice/cloud-platform-environments/blob/main/namespaces/live.cloud-platform.service.justice.gov.uk/create-and-vary-a-licence-api-prod/resources/rds.tf#L13) we use in prod here?

File: run-local.sh [hunk 2]
```diff
@@ -27,12 +27,21 @@ export DB_PASS=create-and-vary-a-licence
...
 # Match with ports defined in docker-compose.yml
-export HMPPS_AUTH_URL=http://localhost:9090/auth
+export HMPPS_AUTH_URL=https://sign-in-dev.hmpps.service.justice.gov.uk/auth
 
 # Make the connection without specifying the sslmode=verify-full requirement
 export SPRING_DATASOURCE_URL='jdbc:postgresql://${DB_SERVER}/${DB_NAME}'
 
+# Bring up the licences-db container needed for the application
+docker-compose down --remove-orphans && docker-compose pull
+docker-compose -f docker-compose.yml up -d
```
Comment (andrewrlee) [inline run-local.sh:37]: should we use `docker compose` now? 
don't need to specify file as that is the default

Comment (gurps-singh) [reply run-local.sh:37]: Doh - it's not as though I just posted about this very thing a couple of hours earlier!

Comment (andrewrlee) [reply run-local.sh]: meh... do

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**